Cargo.toml

```
@@ -18,11 +18,12 @@ keywords = ["data-structure", "no_std"]
 categories = ["data-structures", "no-std"]
 
 [dependencies]
-serde = { version = "1.0", optional = true, features = ["derive"] }
+serde = { version = "1.0", optional = true, default-features = false, features = ["alloc", "derive"] }
 
 [features]
 default = ["use_std"]
 use_std = []
+std = ["use_std"]
```

If we're going to bother with "std", I'd rather make it the primary feature, i.e.
```suggestion
default = ["std"]
std = []
use_std = ["std"] # deprecated alias
```
... and update all the references in `src/lib.rs` and the readme.

---

changed in ed5c626